docs/GettingStarted.md

```diff
@@ -18,23 +18,12 @@ To run ORT the following tools are required:
 * Java (version >= 8)
 * Git
```

Comment:
This should also be removed, as it is now mentioned below.

Reply:
Done.